src/scripts/eicrecon-this.sh.in

```diff
@@ -22,7 +22,10 @@ fi
 #----------------- ROOT
 if [[ -f @ROOT_BINDIR@/thisroot.sh ]] ; then
     unset ROOTSYS  # prevent ROOT from removing /usr/local/bin from PATH
-    source @ROOT_BINDIR@/thisroot.sh
+    # zsh and latest eic-shell require "cd where/root/is" before calling ". bin/thisroot.sh"
```

Comment:
We should correct this incorrect information. It is due to qemu and has nothing to do with zsh or eic-shell. Add a link to the root bug report.